### DISCUSSION TITLE
Commander Pro Zero RPM vs Commander Core XT Zero RPM

### DISCUSSION THREAD

**Discussion by zeldasdad** [EvanMulawski/FanControl.CorsairLink#34] [General]
I've got a commander pro, with a commander core xt daisy-chained.
Corsair ICue uninstalled, HWINFO is running, with the Safety > CorsairLink and Asetek Support unchecked.

5 fans plugged into the pro
6 fans plugged into the core xt

All fans are QL120

At a manual 0 RPM setting, all fans in the pro come to a stop.
At a manual 0 RPM setting, all fans in the core xt run at some minimal RPM (non-zero).

Is this an sdk limitation per device? Hardware limitation? Just trying to understand the delta. I know the class files are (obviously) different for each device.

Thoughts? Thanks!

**Comment by EvanMulawski** [comment]
Hi @tiltedkilt, it is my understanding that the Commander PRO will actually cut power to the fan when set to 0% but the CORE and CORE XT will not. Additionally, some fans will not stop even when the PWM signal is set to 0% and will continue to run at their minimum-rated RPM. For example, if I connect a Noctua fan and a Corsair LL fan to my CORE XT, the Noctua will stop when set to 0% but the Corsair fan will continue to run at its minimum RPM. Does this help?

**Comment by zeldasdad** [comment]
It helps immensely! Thank you so much, and thank you for writing such an excellent tool.
